src/test/java/org/springframework/integration/aws/inbound/SnsInboundChannelAdapterTests.java

```diff
@@ -133,7 +133,7 @@ public void testUnsubscribe() throws Exception {
 		this.mockMvc.perform(
 				post("/mySampleTopic")
 						.header("x-amz-sns-message-type", "UnsubscribeConfirmation")
-						.contentType(MediaType.APPLICATION_JSON)
+						.contentType(MediaType.TEXT_PLAIN)
```

Comment:
I think we should leave of the test to provide `MediaType.APPLICATION_JSON` for consistency.
Even if AWS doesn't support it yet, that doesn't mean that they won't change things in the future.
And we will be ready for them already :smile: